src/ExportProcessors.mss

```diff
@@ -822,4 +822,76 @@ function ProcessSymbol (sobj) {
         }
 
     }
+} //$end
+
+function ProcessActiveSpanners (bobj, meiId) {
+    //$module(ExportProcessors.mss)
+    /*
+        This function does two things:
+         * Adds the bobj's ID to any active spanner's plist.
+         * When we've gone beyond the end of an active spanner, adds 
+           start/end/plist/duration info.
+        This function is also called with `nil` arguments when we've 
```

Comment:
The original motivation for this was that it's not necessarily straightforward to find out whether a note is affected by an `<octave>` element. An explicit `@plist` makes this crystal clear.

`@endid`/`@tstamp2` is very sensible for many types of line like elements. `@endid` can only be added after we've processed the element a line ends on. That's why this commit keeps track of "active spanners".